code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -35,11 +35,11 @@
 		if(VEST_STEALTH)
 			mode = VEST_COMBAT
 			DeactivateStealth()
-			armor = combat_armor
+			armor = getArmor(combat_armor)
```

Comment:
Why not just set both of these in `Initialize`?

Reply:
Not sure what you're asking - do you mean keeping a reference to both combat and stealth armor datums in the object definition to avoid calling `getArmor` every time we wish to switch the vest's mode?

Reply:
as in having

```

/obj/item/clothing/suit/armor/abductor/vest/Initialize(mapload)
	. = ..()
	stealth_armor = getArmor(stealth_armor)
	combat_armor = getArmor(combat_armor)
```

Reply:
Yep sure, done